File: Java/Notebook.java
public class NotebookBits{
	public static void main(String[] args) {
		int x = new java.util.Scanner(System.in).nextInt();
		int total = 0;
		for (int i = 1, j = 0; j < x; j++){
			if (j % 2 != 0) i++;
			total += i;
		}
		System.out.println(total);
	}
}

File: C++/Notebook.cpp
#include <iostream>

int main(){
	int x = 0, total = 0;
	std::cin >> x;
	for (int i = 1, j = 0; j < x; j++){
		if (j % 2 != 0) i++;
		total += i;
	}
	std::cout << total << std::endl;
}